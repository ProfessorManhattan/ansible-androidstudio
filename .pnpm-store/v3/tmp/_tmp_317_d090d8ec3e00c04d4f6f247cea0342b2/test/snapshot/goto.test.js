const parser = require("../main");

describe("goto", function () {
  it("simple", function () {
    expect(parser.parseEval('goto a; echo "Foo";')).toMatchSnapshot();
    expect(parser.parseEval('goto longName; echo "Foo";')).toMatchSnapshot();
  });
});
