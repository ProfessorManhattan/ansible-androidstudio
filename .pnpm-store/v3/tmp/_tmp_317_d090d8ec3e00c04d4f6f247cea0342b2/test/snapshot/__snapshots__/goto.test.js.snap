// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`goto simple 1`] = `
Program {
  "children": Array [
    Goto {
      "kind": "goto",
      "label": Identifier {
        "kind": "identifier",
        "name": "a",
      },
    },
    Echo {
      "expressions": Array [
        String {
          "isDoubleQuote": true,
          "kind": "string",
          "raw": "\\"Foo\\"",
          "unicode": false,
          "value": "Foo",
        },
      ],
      "kind": "echo",
      "shortForm": false,
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;

exports[`goto simple 2`] = `
Program {
  "children": Array [
    Goto {
      "kind": "goto",
      "label": Identifier {
        "kind": "identifier",
        "name": "longName",
      },
    },
    Echo {
      "expressions": Array [
        String {
          "isDoubleQuote": true,
          "kind": "string",
          "raw": "\\"Foo\\"",
          "unicode": false,
          "value": "Foo",
        },
      ],
      "kind": "echo",
      "shortForm": false,
    },
  ],
  "errors": Array [],
  "kind": "program",
}
`;
